tools/runner

```diff
@@ -0,0 +1,42 @@
+#!/bin/env bash
```

Comment:
This should be a Python script I think....

Reply:
It was just extracted from the makefile, so making it a bash script was easier. However: https://github.com/SymbiFlow/sv-tests/pull/11